pkg/server/validate_test.go

```diff
@@ -0,0 +1,59 @@
+package server
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestValidateRequest(t *testing.T) {
+	tt := []struct {
+		testValue     string
+		value         loginRequest
+		expectedError error
+	}{
+		{
+			testValue:     "spaces only",
+			value:         loginRequest{Username: "   ", Password: "    "},
```

Comment:
Please, add tests with a single space in the middle, e.g
username: "sasha yak"
psw: "asasasasas"